scripts/testpypi-smoke.py

```
@@ -88,6 +88,11 @@ class SmokeApp(App):
         class Meta:
             label = "smoke"
             exit_on_close = False
+            # Do NOT inherit this wrapper script's sys.argv — App.run() parses
+            # argv[1:] by default, and the version argument passed to THIS
+            # script ('3.0.15') would be rejected by the app's argparse
+            # ("unrecognized arguments").
+            argv: list[str] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)